test/necb/unit_tests/tests/test_necb_activities.rb

```diff
@@ -6,86 +6,99 @@
 # Checks if BTAP::Activity instances are correctly deployed within BTAP.
 class NECB_Activity_Tests < Minitest::Test
   def test_necb_activities()
+    # ROUND 1 Testing: BTAP data file matching.
+    necb2011_dir = 'lib/openstudio-standards/btap/'
 
-    # File paths.
-    @output_folder = File.join(__dir__, 'output/test_necb_activities')
-    @expected_results_file = File.join(__dir__, '../expected_results/necb_activities_expected_results.json')
-    @test_results_file = File.join(__dir__, '../expected_results/necb_activities_test_results.json')
-    @sizing_run_dir = File.join(@output_folder, 'sizing_folder')
-    @test_results_array = [] # test results storage array
+
+    # ROUND 2 Testing: BTAP runs. BTAP file paths.
+    outd = "output/test_necb_activities"
+    eres = "../expected_results/necb_activities_expected_results.json"
+    tres = "../expected_results/necb_activities_test_results.json"
+    sizd = "sizing_folder"
+
+    @output_folder         = File.join(__dir__, outd)
+    @expected_results_file = File.join(__dir__, eres)
+    @test_results_file     = File.join(__dir__, tres)
+    @sizing_run_dir        = File.join(@output_folder, sizd)
+    @test_results_array    = []
 
     # Intial test condition.
     @test_passed = true
 
     # Range of test options.
     @templates = [
-      'NECB2011',
-    # 'NECB2015',
-    # 'NECB2017'
+      "NECB2011",
+      # "NECB2015",
+      # "NECB2017",
+      # "NECB2020"
     ]
 
-    @epws            = {}
-    @epws['Calgary'] = 'CAN_AB_Calgary.Intl.AP.718770_CWEC2020.epw'
+    @epws = ["CAN_AB_Calgary.Intl.AP.718770_CWEC2020.epw"]
 
     @buildings = [
-      # 'FullServiceRestaurant',
-      # 'HighriseApartment',
-      # 'Hospital',
-      # 'LargeHotel',
-      # 'LargeOffice',
-      # 'MediumOffice',
-      # 'MidriseApartment',
-      # 'Outpatient',
-      # 'PrimarySchool',
-      # 'QuickServiceRestaurant',
-      # 'RetailStandalone',
-      # 'SecondarySchool',
-      # 'SmallHotel',
+      'FullServiceRestaurant',
+      'HighriseApartment',
+      'Hospital',
+      'LargeHotel',
+      'LargeOffice',
+      'LEEPMidriseApartment',
+      'LEEPPointTower',
+      'LEEPTownHouse',
+      'LEEPMultiTower',
```

Comment:
Most of the test OSMs here hold a valid _standards_ building type, which the solution is able to rely on to determine building-wide _activity_ & _category_. The LEEP models do not hold such entries - the solution instead determines valid activity/category pairs based on the preponderance of spacetypes (floor area-weighted).